alicloud/service_alicloud_mns.go

```diff
@@ -21,3 +25,64 @@ func (client *AliyunClient) MnsTopicManager() (ali_mns.AliTopicManager, error) {
 	topicManager := ali_mns.NewMNSTopicManager(*mnsClient)
 	return topicManager, nil
 }
+
+func (client *AliyunClient) MnsSubscriptionManager(topicName string) (ali_mns.AliMNSTopic, error) {
+
+	mnsClient, err := client.Mnsconn()
+	if err != nil {
+		return nil, err
+	}
+	subscriptionManager := ali_mns.NewMNSTopic(topicName, *mnsClient)
+	return subscriptionManager, nil
+}
+
+func ConvertToNotifyStrategyType(notifyStrategyTypeStr string) ali_mns.NotifyStrategyType {
+	var notifyStrategy ali_mns.NotifyStrategyType
+	switch notifyStrategyTypeStr {
+	case "BACKOFF_RETRY":
+		notifyStrategy = ali_mns.BACKOFF_RETRY
+	case "EXPONENTIAL_DECAY_RETRY":
+		notifyStrategy = ali_mns.EXPONENTIAL_DECAY_RETRY
+	default:
+		panic("please input BACKOFF_RETRY or EXPONENTIAL_DECAY_RETRY for notify_strategy")
+	}
+	return notifyStrategy
+}
+
+func ConvertToNotifyContentFormatType(NotifyContentFormatStr string) ali_mns.NotifyContentFormatType {
+	var notifyContentFormat ali_mns.NotifyContentFormatType
+	switch NotifyContentFormatStr {
+	case "XML":
+		notifyContentFormat = ali_mns.XML
+	case "SIMPLIFIED":
+		notifyContentFormat = ali_mns.SIMPLIFIED
+	default:
+		panic("please input XML or SIMPLIFIED for notify_content_format")
+	}
+	return notifyContentFormat
+}
+
+func GetTopicNameAndSubscriptionName(subscriptionId string) (string, string) {
+	arr := strings.Split(subscriptionId, COLON_SEPARATED)
+	return arr[0], arr[1]
+}
+
+func SubscriptionNotExistFunc(err error) bool {
+	if strings.Contains(err.Error(), SubscriptionNotExist) {
+		return true
+	}
```

Comment:
I think just `return strings.Contains(err.Error(), SubscriptionNotExist)` is ok.